### ISSUE TITLE
Running Proxy uses 100% CPU

### ISSUE THREAD

**Issue by ghost** [jolocom/webid-proxy#7]
```
$ java -version

java version "1.8.0_77"                                                                                                                       │
Java(TM) SE Runtime Environment (build 1.8.0_77-b03)                                                                                          │
Java HotSpot(TM) 64-Bit Server VM (build 25.77-b03, mixed mode)
```

**Comment by ghost** [comment]
I updated my java version to `javac 1.8.0_92` which fixed the problem.

**Comment by ghost** [comment]
sorry I have to reopen. Once I query the server cpu load  goes back to 100% :-(

**Comment by ghost** [comment]
when I start mvn in debug mode (-X) I get a loop that seems to cause the CPU load:

`[DEBUG] Selector loop waiting on select
[DEBUG] Selector loop woken up from select, 1/1 selected
[DEBUG] Processing, state UPDATED for SelectChannelEndPoint@60d6fbc5{/0:0:0:0:0:0:0:1:33683<->8111,Open,ISHUT,out,R,-,0,HttpConnection}{io=1,kio=1,
kro=1}
[DEBUG] onSelected 1->0 for SelectChannelEndPoint@60d6fbc5{/0:0:0:0:0:0:0:1:33683<->8111,Open,ISHUT,out,R,-,0,HttpConnection}{io=0,kio=1,kro=1}
[DEBUG] FILL_INTERESTED-->FILLING HttpConnection@6e43b051{FILLING}
[DEBUG] Updating key, state UPDATE_PENDING for SelectChannelEndPoint@60d6fbc5{/0:0:0:0:0:0:0:1:33683<->8111,Open,ISHUT,out,-,-,0,HttpConnection}{io
=0,kio=1,kro=1}
[DEBUG] Key interests update 1 -> 0 for SelectChannelEndPoint@60d6fbc5{/0:0:0:0:0:0:0:1:33683<->8111,Open,ISHUT,out,-,-,0,HttpConnection}{io=0,kio=
1,kro=1}
[DEBUG] Selector loop waiting on select
[DEBUG] HttpConnection@6e43b051{FILLING} onFillable HttpChannelState@65663c03{s=IDLE i=true a=null}
[DEBUG] atEOF HttpParser{s=CLOSED,0 of -1}
[DEBUG] parseNext s=CLOSED HeapByteBuffer@1[p=0,l=0,c=0,r=0]={<<<>>>}
[DEBUG] fillInterested HttpConnection@6e43b051{FILLING}
[DEBUG] FILLING-->FILLING_FILL_INTERESTED HttpConnection@6e43b051{FILLING_FILL_INTERESTED}
[DEBUG] FILLING_FILL_INTERESTED-->FILL_INTERESTED HttpConnection@6e43b051{FILL_INTERESTED}
[DEBUG] Changing interests in state UPDATED for SelectChannelEndPoint@60d6fbc5{/0:0:0:0:0:0:0:1:33683<->8111,Open,ISHUT,out,R,-,0,HttpConnection}{i
o=0,kio=0,kro=1}
[DEBUG] changeInterests s=UPDATED 0->1 for SelectChannelEndPoint@60d6fbc5{/0:0:0:0:0:0:0:1:33683<->8111,Open,ISHUT,out,R,-,0,HttpConnection}{io=0,k
io=0,kro=1}
[DEBUG] Queued change org.eclipse.jetty.io.SelectChannelEndPoint$1@172caffc
[DEBUG] Selector loop woken up from select, 0/1 selected
[DEBUG] Running change org.eclipse.jetty.io.SelectChannelEndPoint$1@172caffc
[DEBUG] Updating key, state UPDATE_PENDING for SelectChannelEndPoint@60d6fbc5{/0:0:0:0:0:0:0:1:33683<->8111,Open,ISHUT,out,R,-,0,HttpConnection}{io
=1,kio=0,kro=1}
[DEBUG] Key interests update 0 -> 1 for SelectChannelEndPoint@60d6fbc5{/0:0:0:0:0:0:0:1:33683<->8111,Open,ISHUT,out,R,-,0,HttpConnection}{io=1,kio=
0,kro=1}
`

**Comment by peacekeeper** [comment]
this may have been fixed with commit 7a8fcd00f0db84b23c740f86ff196f6d25c9ec4d (use newer jetty version), can you test?

**Comment by ghost** [comment]
Yes it looks good indeed. Thank you very much! ;-)
